File: data/steps-codecomplete/src/test/java/StepImplementation.java
import com.thoughtworks.gauge.Step;
import com.thoughtworks.gauge.Table;
import com.thoughtworks.gauge.TableRow;

import java.util.HashSet;

import static org.assertj.core.api.Assertions.assertThat;

public class StepImplementation {
    @Step("implemented step one")
    public void implementedStepOne()
    {
    }

    @Step("implemented step two")
    public void implementedStepTwo()
    {
    }
}

File: data/steps-codecomplete/impl/StepImplementation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Gauge.CSharp.Lib;
using Gauge.CSharp.Lib.Attribute;

namespace DotNet.Template
{
    public class StepImplementation
    {
        [Step("implemented step one")]
        public void implementedStepOne()
        {
        }

        [Step("implemented step two")]
        public void implementedStepTwo()
        {
        }
    }
}
